package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "po-gettext-loader",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Not sure should bump the major or minor version, please help to review.

Reply:
Let it be 1.0.0 :)